.github/workflows/ci.yaml

```diff
@@ -29,6 +29,8 @@ jobs:
   ImageBuildAndPush:
     name: Build Image and Push
     uses: uc-cdis/.github/.github/workflows/image_build_push.yaml@master
+    with:
+      BUILD_PLATFORMS: "linux/amd64"
```

Comment:
not building linux/arm64 isn't great for people with m1 macs trying to run images locally - is this a recommendation from the SDET or PE teams?

Reply:
It was not, I got mixed up with the thread regarding  `_native.yaml. Will change it to that.